build.gradle.kts

```diff
@@ -22,6 +22,7 @@ repositories {
     mavenCentral()
     maven("https://maven.scijava.org/content/groups/public")
     maven("https://jitpack.io")
+    maven("https://raw.githubusercontent.com/kotlin-graphics/mary/master")
```

Comment:
Can we please get artifacts deployed to maven.scijava.org, instead of a custom GitHub-based repository, which [should not be done](https://stackoverflow.com/a/22210816/1207769)?

Reply:
What shouldn't be done is loading big jars (ie fat jars)

This is not the case, the ones I push weight few KBs

Reply:
It shouldn't be done at all IMHO—now you have to run that repo forever or else old builds will break. And as described in that article, it creates a situation where there's another repository that has to be queried for all the dependencies, slowing down builds. Just release the projects on maven.scijava.org or oss.sonatype.org! Once you have the infrastructure in place for Sonatype, deploying is a button push—the only huge PITA is getting it set up at the beginning.

Reply:
> now you have to run that repo forever or else old builds will break. 

True, but you know what I have to do to run it forever? Just avoid deleting it

I might be move everything to maven, though ([here](https://github.com/kotlin-graphics/glm/issues/35))

I'd love to deploy to maven.scijava.org, but I have a dev way to release very often and that wasn't compatible with that, or do I recall it wrong?

Reply:
Publishing to maven.scijava.org is just a standard deploy over WebDAV, I'm sure the Gradle tooling will support it with your credentials both manually and/or automatically. As long as your tooling doesn't try to deploy the same release artifact more than once, which is forbidden due to release artifacts being immutable.

Reply:
How much does it take the publication over there? How long before it's available for consumption?

Reply:
Artifacts published to maven.scijava.org are available instantly. Artifacts published to oss.sonatype.org (or s01.oss.sonatype.org) are available instantly from those URLs, but typically take 10-30 minutes to get mirrored over to Maven Central.